src/plugins/timelist/ExpandedViewItem.vue

```diff
@@ -122,47 +122,60 @@ const INFERRED_EXECUTION_STATES = {
 export default {
   inject: ['openmct', 'domainObject', 'path'],
   props: {
-    item: {
-      type: Object,
-      required: true
+    name: {
+      type: String,
+      default: ''
+    },
+    start: {
+      type: Number,
+      default: 0
+    },
+    end: {
+      type: Number,
+      default: 0
+    },
+    duration: {
+      type: Number,
+      default: 0
+    },
+    countdown: {
+      type: Number,
+      default: 0
+    },
+    cssClass: {
+      type: String,
+      default: ''
```

Comment:
This is great! Do you think name, start, end, duration, and countdown, should be required props?